### DISCUSSION TITLE
Promoting OpenStreetMap projects with MapScaping - do we want to apply?

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#4458] [Q&A] [answered]
See https://blog.opencagedata.com/post/partnering-with-mapscaping-to-promote-openstreetmap-projects

**Comment by CalliOSM** [comment]
Good idea. I think StreetComplete has a good chance to win this. Does anyone know the combined reach of both podcasts?

**Comment by matkoniecz** [comment, marked as answer]
out of time for that specific activity
